index.html

```diff
@@ -702,9 +698,9 @@
               </li>
               <li>If the orientation change was triggered by a user gesture
               such as the user turning the device, as opposed to a call to
-              <a data-link-for='ScreenOrientation'>lock</a>, the <a>task</a>
-              MUST be annotated with <code>process user orientation change
-              </code> when running the next step.
+              <a>lock</a>, the <a>task</a> MUST be annotated with
+              <code>process user orientation change</code> when running the
```

Comment:
Nit: this `code` should probably have been an `<a>`

Reply:
Fixed all comments except this one, as it needs definition or algo for `process user orientation change` term.